log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -201,6 +217,8 @@ public void configure(LoggingConfiguration config)
             disableConsole();
         }
 
+        setBuiltInLevels();
```

Comment:
shouldn't built in levels be loaded before user log levels?

Reply:
They should. Aren't they?